NOTICE.txt

```diff
@@ -547,6 +547,39 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+--------------------------------------------------------------------
+Dependency: github.com/fsnotify/fsevents^
```

Comment:
Any idea why this appears twice in the file? What's with the `fsevents^`?

Reply:
My mistake